Reply/app/src/main/res/values/themes.xml

```diff
@@ -83,8 +69,8 @@
         <item name="motionInterpolatorOutgoing">@android:interpolator/accelerate_cubic</item>
 
         <!--Styles-->
-        <item name="bottomAppBarStyle">@style/Widget.MaterialComponents.BottomAppBar.PrimarySurface</item>
-        <item name="bottomSheetStyle">@style/Widget.MaterialComponents.BottomSheet.Modal</item>
+        <item name="bottomAppBarStyle">@style/Widget.Material3.BottomAppBar</item>
+        <item name="bottomSheetStyle">@style/Widget.Material3.BottomSheet.Modal</item>
```

Comment:
are these MaterialComponent -> Material3 widget changes part of this PR? (hard to tell because this PR includes the other one) if so let's make the PR description mention both Material3 typography and component updates

Reply:
Alright. just updated.